Makefile

```diff
@@ -100,8 +76,9 @@ $(BINDIR)/deepcopy-gen: vendor
 ###############################################################################
 # Static checks
 ###############################################################################
-.PHONY: static-checks
-static-checks: check-format check-gen-files
+# TODO: re-enable all linters
+LINT_ARGS += --disable gosimple,unused,structcheck,errcheck,deadcode,varcheck,ineffassign,staticcheck,govet
+LOCAL_CHECKS = goimports check-gen-files
```

Comment:
How will this be used?

Reply:
Both of those vars are used in `static-checks` defined in Makefile.common. I'm disabling those linters for now because they're flagging a bunch of errors and fixing them is out of scope for this PR.